.rspec-local

```diff
@@ -1 +0,0 @@
---format documentation
```

Comment:
can you get rid of this commit/change? perhaps do a git checkout of it from develop branch.